commons/src/main/java/com/powsybl/commons/json/JsonUtil.java

```diff
@@ -582,6 +582,29 @@ public static String getSourceVersion(DatabindContext context, String sourceVers
         return context.getAttribute(sourceVersionAttributeKey) != null ? (String) context.getAttribute(sourceVersionAttributeKey) : null;
     }
 
+    public static <T> T readValueWithContext(DeserializationContext context, JsonParser parser, Class<?> type) {
```

Comment:
We could just add a short javadoc to explain the added value.

Note that we could have given the responsibility of handling the `null` values to our deserializers instead, but the change would be more painful, I think it's better this way :)

Reply:
@HugoKulesza : I will just rename to simpler `readValue`, before merging